TerminalDocs/customize-settings/key-bindings.md

```diff
@@ -196,14 +193,13 @@ This allows you to enter "focus mode", which hides the tabs and titlebar.
 _This command is not currently bound in the default settings_.
 
 ```json
-{ "command": "toggleFocusMode" }
+{ "command": "toggleFocusMode", "keys": "" }
```

Comment:
this is not technically true. i guess it's helpful to let users know to bind it?

Reply:
Yeah I figured it just helps with structure?